src/main/java/frc/robot/subsystems/instances/Lift.java

```diff
@@ -7,24 +7,119 @@
 
 package frc.robot.subsystems.instances;
 
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+import edu.wpi.first.wpilibj.Compressor;
+import edu.wpi.first.wpilibj.Solenoid;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.subsystems.interfaces.LiftInterface;
 
 public class Lift extends SubsystemBase implements LiftInterface {
   /**
    * Creates a new Lift.
    */
+  private static WPI_TalonSRX leftMotorLeader;
+  private static WPI_TalonSRX rightMotorLeader;
+  private static WPI_TalonSRX leftMotorFollower;
+  private static WPI_TalonSRX rightMotorFollower;
+  private static Compressor compressor;
```

Comment:
I dont think we want to control the compressor from this subsystem. We should probably remove.